libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/room/RustMatrixRoom.kt

```diff
@@ -187,7 +187,13 @@ class RustMatrixRoom(
         _membersStateFlow.value = MatrixRoomMembersState.Pending(prevRoomMembers = currentMembers)
         var rustMembers: List<RoomMember>? = null
         try {
-            rustMembers = innerRoom.members()
+            rustMembers = buildList {
+                while (true) {
+                    // Loading the whole iterator as a stop-gap measure.
+                    // We should probably implement some sort of paging in the future.
+                    addAll(innerRoom.members().nextChunk(100u) ?: break)
```

Comment:
Should we put a higher chunk size for now?

Reply:
1000 is ok?

Reply:
I guess!